### ISSUE TITLE
Inconsistent repsonse from rhandsontable::rHandsontableOutput()

### ISSUE THREAD

**Issue by veer-marwah** [jrowen/rhandsontable#217]
Hi,

rhandsontable::rHandsontableOutput() is performing inconsistently in my implementation and tests. When the user interacts with the rendered the table in the ui the data from the rendered table is passed to the server in input (as intended), but when the user does not interact with the rendered table, the server gets NULL as input for the rendered table. So the data from the rendered table is not passed to the server unless the user interacts with it.

```
library(shiny)
library(rhandsontable)

ui = shinyUI(fluidPage(
	titlePanel("Test rhandsontable"),
	sidebarLayout(
		sidebarPanel(
			h1("Sidebar!")
		),
		mainPanel(
			rHandsontableOutput("RH"),
			br(),
			actionButton("test_submit", "Test Rendered Table")
		)
	)
))

server = function(input, output) {
	output$RH <- renderRHandsontable({
		DF <- data.frame(val = 1:10, bool = TRUE, big = LETTERS[1:10],
			small = letters[1:10],
			dt = seq(from = Sys.Date(), by = "days", length.out = 10),
			stringsAsFactors = FALSE
		)
		rhandsontable(DF, rowHeaders = NULL, width = 550, height = 300) %>%
		hot_col(col = "big", type = "dropdown", source = LETTERS) %>%
		hot_col(col = "small", type = "autocomplete", source = letters, strict=FALSE)
	})
	
	observeEvent(input$test_submit, {
		print("Testing RH...")
		print(is.null(input$RH))
		print(str(input$RH))
		print(class(input$RH))
		RH <- rhandsontable::hot_to_r(input$RH)
		print(str(RH))
		print("Testing Completed!")
        })
}
```

I hope you are able to replicate the issue. My sessionInfo() is provided below.

```
> sessionInfo()
R version 3.4.3 (2017-11-30)
Platform: x86_64-pc-linux-gnu (64-bit)

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
[1] rhandsontable_0.3.5 shiny_1.0.5        

loaded via a namespace (and not attached):
 [1] htmlwidgets_0.9 compiler_3.4.3  magrittr_1.5    R6_2.2.2       
 [5] tools_3.4.3     htmltools_0.3.6 yaml_2.1.16     Rcpp_0.12.14   
 [9] jsonlite_1.5    digest_0.6.13   xtable_1.8-2    httpuv_1.3.5   
[13] mime_0.5

```

Regards,

Veer

**Comment by jrowen** [comment]
I think this is the same as #216. Can you try the dev version and let us know if it's working as expected?

**Comment by veer-marwah** [comment]
Hi @jrowen,

I updated my version of rhandsontable and I am not experiencing this issue anymore. Thank you.

```
packageVersion("rhandsontable")
[1] ‘0.3.5.5’
```

Regards,

Veer

**Comment by jrowen** [comment]
Fixed in v0.3.6.  Thanks for reporting.
